src/App.tsx

```diff
@@ -15,36 +17,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const sortingActions = {
+  alphabetically: 'Alphabetically',
+  length: 'Length',
+};
+
+function handleGoodsSortAction(
+  goods: string[],
+  action: string,
+  isReversed: boolean,
+) {
+  const renderedGoods = [...goods];
+
+  renderedGoods.sort((good1, good2) => {
+    switch (action) {
+      case sortingActions.alphabetically:
+        return good1.trim().localeCompare(good2.trim());
+      case sortingActions.length:
+        return good1.trim().length - good2.trim().length;
+      default:
+        return 0;
+    }
+  });
+
+  if (isReversed) {
+    renderedGoods.reverse();
+  }
+
+  return renderedGoods;
+}
```

Comment:
[CODE STYLE] - The function's name 'handleGoodsSortAction' should begin with a verb to clearly indicate what it does. Consider renaming it to something like 'sortGoods' to better reflect its purpose.

Reply:
no intel to fix this in object